tests/test_module_seletopclusts.py

```diff
@@ -360,30 +341,34 @@ def test_select_top_sized_clusts_models(clustered_models):
             ind += 1
 
 
-def test_sort_models_unranked(unranked_models):
+def test_sort_models_unranked(protprot_input_list):
     """Test sorting of unranked models."""
-    sorted_models, _note = sort_models(unranked_models)
-    assert sorted_models == unranked_models
+    sorted_models, _note = sort_models(protprot_input_list)
+    assert sorted_models == protprot_input_list
     assert type(_note) == str
 
 
 def test_sort_models_ranked(ranked_models):
     """Test sorting of ranked models."""
     sorted_models, _note = sort_models(ranked_models)
     assert not _note
-    assert sorted_models[0].file_name == "protprot_complex_2.pdb"
-    assert sorted_models[1].file_name == "protprot_complex_1.pdb"
+    assert sorted_models[0].file_name == ranked_models[1].file_name
+    assert sorted_models[1].file_name == ranked_models[0].file_name
+
+    # assert sorted_models[0].file_name == "protprot_complex_2.pdb"
+    # assert sorted_models[1].file_name == "protprot_complex_1.pdb"
```

Comment:
can we remove this? or do you want to keep the commented lines for future reference?